examples/make_eeg_bids.py

```diff
@@ -0,0 +1,181 @@
+events = find_events(raw)
+
+###############################################################################
+# Now we have everything to start a new BIDS directory using our data.
+# To do that, we can use the high level function `raw_to_bids`, which is the
+# core of MNE-BIDS. Generally, `raw_to_bids` tries to extract as much meta data
+# as possible from the raw data and then format it in a BIDS compatible way.
+# `raw_to_bids` takes a bunch of inputs, most of which are however optional.
+# The required inputs are described in the docstring:
+#
+# .. code-block:: python
+#    """
+#    Walk over a folder of files and create BIDS compatible folder.
+#
+#    Parameters
+#    ----------
+#    subject_id : str
+#        The subject name in BIDS compatible format ('01', '02', etc.)
+#    task : str
+#        Name of the task the data is based on.
+#    raw_file : str | instance of mne.Raw
+#        The raw data. If a string, it is assumed to be the path to the raw
+#        data file. Otherwise it must be an instance of mne.Raw
+#    output_path : str
+#        The path of the BIDS compatible folder
+#     """
+# Let's assume the previous file name `sub-05_task-matchingpennies_eeg`
+# contained an error and that this was actually `sub-01` and we also want to
+# have another task name `MatchingPennies`.
+subject_id = '01'
+task = 'MatchingPennies'
+raw_file = raw  # We can specify the raw file, or the path to the .vhdr file.
+output_path = _TempDir()
+
+###############################################################################
+# Now we just need to specify a few more EEG details to get something sensible:
+
+# First, tell `MNE-BIDS` that it is working with EEG data:
+kind = 'eeg'
+
+# The EEG reference used for the recording
+eeg_reference = 'Fz'
+
+# brief description of the event markers present in the data. This will become
+# the `trial_type` column in our BIDS `events.tsv`.
+event_id = {'left': 1, 'right': 2}
+
+# Now convert our data to be in a new BIDS dataset.
+raw_to_bids(subject_id=subject_id, task=task, raw_file=raw_file,
+            output_path=output_path, kind=kind,
+            eeg_reference=eeg_reference, event_id=event_id)
+
+###############################################################################
+# Step 3: Check and compare with standard
+# ---------------------------------------
+# Now we have written our BIDS directory. Having read the BIDS specification,
+# we know that the structure should approximately look like this:
+#
+# .. code-block:: python
+#
+#    ---------- CHANGES
+#    ---------- dataset_description.json
+#    ---------- participants.tsv
+#    ---------- README
+#    ---------- sub-xx
+#        ---------- modality
+#            ---------- sub-xx_task-yy_events.tsv
+#            ---------- sub-xx_task-yy_modality.ext
+#            ---------- sub-xx_task-yy_modality.json
+#
+#
+# Now, what does it actually look like?
+for root, dirs, files in os.walk(output_path):
```

Comment:
should be a function since you duplicate it

Reply:
in fact, you could probably add it as a public function in `mne-bids`. It's a nice way to visualize the folders